src/Features/LanguageServer/Protocol/Handler/RequestContext.cs

```diff
@@ -25,9 +34,25 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler
         public string? ClientName { get; }
 
         public RequestContext(ClientCapabilities clientCapabilities, string? clientName)
+            : this(null, null, clientCapabilities, clientName)
```

Comment:
1. named params for null.
2. my preference is that the common params be at teh same location.  so it would be `ClientCap, clientName, solution, solutionUpdated)

Reply:
This is cleaned up in a follow up PR